index.html

```diff
@@ -41,7 +41,7 @@
             <h2>Installing DEER</h2>
             <p class="col-12">To install the script on your project, use the <code>deer.js</code> on your page.</p>
             <small class="col-12">
-                <pre>&lt;script src="//centerfordigitalhumanities.github.io/deer/releases/alpha-0.11/deer.js" type="module">&lt;/script></pre>
+                <pre>&lt;script src="//deer.rerum.io/DEER/releases/alpha-0.11/deer.js" type="module">&lt;/script></pre>
```

Comment:
Oh, nice!  Are we going to make this our CDN?  Like can I do this in the deer-lr code?

Reply:
I think that is reasonable. It is how we tell people to link it on the site. Also, we will need to remember to put it up on other CDNs when we hit 1.0 for realz.